capa/v31.0.0/release.yaml

```
@@ -0,0 +1,139 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: aws-31.0.0
+spec:
+  apps:
+  - name: aws-ebs-csi-driver
+    version: 3.0.5
+    dependsOn:
+    - cloud-provider-aws
+  - name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+    dependsOn:
+    - prometheus-operator-crd
+  - name: aws-nth-bundle
+    version: 1.2.1
+  - name: aws-pod-identity-webhook
+    version: 1.19.1
+    dependsOn:
+    - cert-manager
+  - name: capi-node-labeler
+    version: 1.1.0
+  - name: cert-exporter
+    version: 2.9.6
+    dependsOn:
+    - kyverno-crds
+  - name: cert-manager
+    version: 3.9.1
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cert-manager-crossplane-resources
```

```suggestion
  - name: cert-manager-crossplane-resources
    catalog: cluster
```

This is still the case and other `-crossplane-resources` apps are also being pushed there, because this app ends up in the MC, not in the WC.